linux/instructions/step_5.md

```diff
@@ -0,0 +1,26 @@
+Step 5: Fix Networking
```

Comment:
As we have learned recently, help me over-scrutinize these instructions please :)